src/ui/ToolchainSelection.ts

```diff
@@ -150,13 +150,19 @@ async function getQuickPickItems(
         });
     const toolchains = (await SwiftToolchain.getToolchainInstalls())
         .reverse()
-        .map<SwiftToolchainItem>(toolchainPath => ({
-            type: "toolchain",
-            label: path.basename(toolchainPath, ".xctoolchain"),
-            detail: toolchainPath,
-            toolchainPath: path.join(toolchainPath, "usr"),
-            swiftFolderPath: path.join(toolchainPath, "usr", "bin"),
-        }));
+        .map<SwiftToolchainItem>(toolchainPath => {
+            let label = path.basename(toolchainPath, ".xctoolchain");
+            if (label === "swift-latest") {
+                label = "Latest Installed Toolchain";
+            }
+            return {
```

Comment:
When someone selects the latest toolchain, do they expect to be selecting the latest toolchain installed at that point, or the latest installed toolchain ad infinitum. ie If they install another toolchain later on it automatically gets used.
 
Should we point to the linked toolchain or the symbolic link?

Reply:
That's a good question. My intention with this option was the latter: to always point at the most recently installed toolchain via the symbolic link. Otherwise, why have the option in the first place when you can just select the toolchain below it? However, the current wording is potentially confusing.

How does `Always Use Most Recently Installed Toolchain` sound? Seems a bit wordy to me, but it does get the point across.

Reply:
Yeah that is a bit wordy, maybe we can keep as is, but display a info window about it always being the most recently installed toolchain.

Reply:
I've added a notification that pops up after the toolchain has been selected. Let me know if you want to change the wording.